opentelemetry/src/sdk/trace/span_processor.rs

```diff
@@ -294,11 +292,13 @@ impl BatchSpanProcessor {
                                     &delay,
                                     batch,
                                 )
-                                .await,
+                                    .await,
```

Comment:
I think these moves are unintentional, doesn't seem like cargo fmt is working on this part? odd.

Reply:
Couple more of these left, I wonder why cargo fmt doesn't rewrite these..

Reply:
I think I got all of them. It's stange that `cargo fmt` didn't correct those.